lib/stdlib.ngs

```diff
@@ -2105,6 +2105,30 @@ section "Range" {
 	TEST 1 =~ 0..5
 	TEST 0 =~ 0..5
 	TEST 5 !~ 0..5
+
+	doc Ensure whether n is in the range
+	doc %EX - ensure(1, 3..10) #3
+	F ensure(n:Int, r:NumRange){
+		tmp = null
+    	last = 0
+
+    	if r.include_end == false {
+        	null
+    	}
+
+    	tmp = r.map(X*1)
```

Comment:
* Please don't convert to an array. It's a waste of time and memory. It should be O(1) operation, not O(n).
* Please use `.start`, `.end`, `.include_start`, `.include_end`, and `.step` of the Range to calculate the result.